api/internal/handler/upstream/upstream.go

```diff
@@ -270,27 +267,15 @@ func (h *Handler) Exist(c droplet.Context) (interface{}, error) {
 		return nil, err
 	}
 
-	sort := store.NewSort(nil)
-	filter := store.NewFilter([]string{"name", name})
-	pagination := store.NewPagination(0, 0)
-	query := store.NewQuery(sort, filter, pagination)
-	rows := store.NewFilterSelector(toRows(ret), query)
```

Comment:
maybe we could remove these functions now, please confirm whether these functions are invoked elsewhere.

Reply:
I suggest using a new issue to record this problem. Later, I will check whether these methods are used in other places, and submit a new PR to delete them. If my idea is wrong, please correct me.

Reply:
@Jaycean  that's good, thanks.